pkg/apis/autoscaling/v1alpha1/clusterautoscaler_types.go

```diff
@@ -64,4 +64,5 @@ type ScaleDownConfig struct {
 	DelayAfterAdd     string `json:"delayAfterAdd"`
 	DelayAfterDelete  string `json:"delayAfterDelete"`
 	DelayAfterFailure string `json:"delayAfterFailure"`
+	UnneededTime      string `json:"unneededTime"`
```

Comment:
I think the variable should reflect `scale-down`. And it should be optional too.

Reply:
Ah, but this is in `ScaleDownConfig`... ok, forgot my comment.

Reply:
> I think the variable should reflect `scale-down`

I what sense? It's under `ScaleDownConfig` struct.

> And it should be optional too.

Yeah. Agree.